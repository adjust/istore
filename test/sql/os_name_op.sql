BEGIN;
-- os_name_op should implement gt;
-- ./spec/os_name_op_spec.rb:8;
CREATE EXTENSION istore;
SELECT 'ios'::os_name > 'android'::os_name;
SELECT 'ios'::os_name >= 'android'::os_name;
SELECT NOT('ios'::os_name < 'android'::os_name);
SELECT NOT('ios'::os_name <= 'android'::os_name);
ROLLBACK;
BEGIN;
-- os_name_op should implement lt;
-- ./spec/os_name_op_spec.rb:15;
CREATE EXTENSION istore;
SELECT 'ios'::os_name > 'android'::os_name;
SELECT 'ios'::os_name >= 'android'::os_name;
SELECT NOT('ios'::os_name < 'android'::os_name);
SELECT NOT('ios'::os_name <= 'android'::os_name);
ROLLBACK;
BEGIN;
-- os_name_op should implement eq;
-- ./spec/os_name_op_spec.rb:22;
CREATE EXTENSION istore;
SELECT NOT('ios'::os_name = 'android'::os_name);
SELECT 'android'::os_name = 'android'::os_name;
ROLLBACK;
BEGIN;
-- os_name_op should implement neq;
-- ./spec/os_name_op_spec.rb:27;
CREATE EXTENSION istore;
SELECT 'ios'::os_name != 'android'::os_name;
SELECT NOT('android'::os_name != 'android'::os_name);
ROLLBACK;
