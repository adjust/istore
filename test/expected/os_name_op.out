BEGIN;
-- os_name_op should implement gt;
-- ./spec/os_name_op_spec.rb:8;
CREATE EXTENSION istore;
SELECT 'ios'::os_name > 'android'::os_name;
 ?column? 
----------
 t
(1 row)

SELECT 'ios'::os_name >= 'android'::os_name;
 ?column? 
----------
 t
(1 row)

SELECT NOT('ios'::os_name < 'android'::os_name);
 ?column? 
----------
 t
(1 row)

SELECT NOT('ios'::os_name <= 'android'::os_name);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
-- os_name_op should implement lt;
-- ./spec/os_name_op_spec.rb:15;
CREATE EXTENSION istore;
SELECT 'ios'::os_name > 'android'::os_name;
 ?column? 
----------
 t
(1 row)

SELECT 'ios'::os_name >= 'android'::os_name;
 ?column? 
----------
 t
(1 row)

SELECT NOT('ios'::os_name < 'android'::os_name);
 ?column? 
----------
 t
(1 row)

SELECT NOT('ios'::os_name <= 'android'::os_name);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
-- os_name_op should implement eq;
-- ./spec/os_name_op_spec.rb:22;
CREATE EXTENSION istore;
SELECT NOT('ios'::os_name = 'android'::os_name);
 ?column? 
----------
 t
(1 row)

SELECT 'android'::os_name = 'android'::os_name;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
-- os_name_op should implement neq;
-- ./spec/os_name_op_spec.rb:27;
CREATE EXTENSION istore;
SELECT 'ios'::os_name != 'android'::os_name;
 ?column? 
----------
 t
(1 row)

SELECT NOT('android'::os_name != 'android'::os_name);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
